src/patternfly/components/Dropdown/dropdown-toggle.hbs

```diff
@@ -0,0 +1,12 @@
+<button class="pf-c-dropdown__toggle" id="{{id}}" aria-haspopup="true" aria-expanded="{{#if dropdown--IsExpanded}}true{{else}}false{{/if}}"
```

Comment:
can we add `dropdown-toggle--attribute/modifier` and fix the indenting in this file?

Reply:
resolved